plugins/task/privacyconsent/src/Extension/PrivacyConsent.php

```diff
@@ -0,0 +1,257 @@
+ * @subpackage  Task.PrivacyConsent
+ *
+ * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\Task\PrivacyConsent\Extension;
+
+use Joomla\CMS\Application\ApplicationHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Mail\Exception\MailDisabledException;
+use Joomla\CMS\Mail\MailTemplate;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\User\UserHelper;
+use Joomla\Component\Scheduler\Administrator\Event\ExecuteTaskEvent;
+use Joomla\Component\Scheduler\Administrator\Task\Status;
+use Joomla\Component\Scheduler\Administrator\Task\Task;
+use Joomla\Component\Scheduler\Administrator\Traits\TaskPluginTrait;
+use Joomla\Database\DatabaseAwareTrait;
+use Joomla\Database\ParameterType;
+use Joomla\Event\SubscriberInterface;
+use PHPMailer\PHPMailer\Exception as phpmailerException;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * A task plugin. Offers 2 task routines Invalidate Expired Consents and Remind Expired Consents
+ * {@see ExecuteTaskEvent}.
+ *
+ * @since __DEPLOY_VERSION__
+ */
+final class PrivacyConsent extends CMSPlugin implements SubscriberInterface
+{
+    use DatabaseAwareTrait;
+    use TaskPluginTrait;
+
+    /**
+     * @var string[]
+     * @since __DEPLOY_VERSION__
+     */
+    private const TASKS_MAP = [
+        'privacy.consent' => [
+            'langConstPrefix' => 'PLG_TASK_PRIVACYCONSENT_INVALIDATE',
+            'method'          => 'privacyConsents',
+            'form'            => 'privacyconsentForm',
+        ],
+    ];
+
+    /**
+     * @var boolean
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    protected $autoloadLanguage = true;
+
+    /**
+     * @inheritDoc
+     *
+     * @return string[]
+     *
+     * @since __DEPLOY_VERSION__
+     */
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            'onTaskOptionsList'    => 'advertiseRoutines',
+            'onExecuteTask'        => 'standardRoutineHandler',
+            'onContentPrepareForm' => 'enhanceTaskItemForm',
+        ];
+    }
+
+    /**
+     * Method to send the remind for privacy consents renew.
+     *
+     * @param   ExecuteTaskEvent  $event  The `onExecuteTask` event.
+     *
+     * @return integer  The routine exit code.
+     *
+     * @since  __DEPLOY_VERSION__
+     * @throws \Exception
+     */
+    private function privacyConsents(ExecuteTaskEvent $event): int
+    {
+        // Load the parameters.
+        $expire = (int) $event->getArgument('params')->consentexpiration ?? 365;
+        $remind = (int) $event->getArgument('params')->remind ?? 30;
+
+        if (($this->invalidateExpiredConsents($expire) === Status::OK) &&
+            ($this->remindExpiringConsents($expire, $remind) === Status::OK)) {
```

Comment:
```suggestion
        if (
            $this->invalidateExpiredConsents($expire) === Status::OK
            && $this->remindExpiringConsents($expire, $remind) === Status::OK
        ) {
```
Fix PHPCS, hopefully the last one, and remove unnecessary brackets.